### ISSUE TITLE
Introducing FROM and THRU range ( When applicable )  for billing_codes  in-network-rates

### ISSUE THREAD

**Issue by narenlulla** [CMSgov/price-transparency-guide#464]
### Discussed in https://github.com/CMSgov/price-transparency-guide/discussions/405

<div type='discussions-op-text'>

<sup>Originally posted by **iGokul-G** March  1, 2022</sup>
Introducing a "FROM" and "THRU" range for billing_code  might help reduce the record count only when applicable. Payers do use code ranges to represent services  </div>

**Comment by rgouldkc** [comment]
Good idea. We have LOTS of rates that apply to a range of billing codes for many providers. Some of the ranges are rather large.

**Comment by shaselton-usds** [comment]
Closing Issue as implementation isn't taking this route at this point.
